dev/com.ibm.ws.cdi.extension_fat/fat/src/com/ibm/ws/cdi/extension/apps/spi/SPIExtensionServlet.java

```diff
@@ -15,8 +15,13 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
 
+import com.ibm.ws.cdi.extension.tests.CDIExtensionRepeatActions;
```

Comment:
Might be a good idea to split this new CDI test into a separate commit or even a separate PR?

Reply:
It is in a separate commit